src/Bot.ts

```diff
@@ -35,6 +36,9 @@ export class Bot extends builder.UniversalBot {
             // set on "receive" of message
             new SetLocaleFromTeamsSetting(),
 
+            // set on "receive" of Adaptive Card Submit Action
```

Comment:
Let's change this so that there is no comment here; so the new AdaptiveCardSubmitAction(), is right below new SetLocalFromTeamsSetting(),; and let's change the comment '//set on "receive" of message' to '//set on "receive" of incoming payload'

Reply:
Done.